### PR TITLE
Broken Link Pass

### PR DESCRIPTION
**Pull request by esethna**
Using: http://www.deadlinkchecker.com/

### CODE REVIEW

**Comment by it33** [comment]
So... just curious why are we keeping [2.x documentation?](https://github.com/mattermost/docs/blob/e949f58a1160645493dce51f84950e86785e1de6/source/archives/docs-v2.2.md)

It's missing critical security updates, if anything we want people to move to the latest version rather than helping them stay on the out-of-date build...

**Comment by lfbrock** [comment]
@esethna I think we can remove the 2.x documentation now.

@it33 that was back from when we had to do all those large v3.0 changes and you asked for us to keep a link for people still on v2.x.

File: source/install/requirements.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@ Please see `Mattermost Deployment Overview <http://docs.mattermost.com/deploymen
 
 .. figure:: ../images/network.PNG
    :alt: image
-`View Mattermost Network Diagram <../images/network.PNG>`__
+`View Mattermost Network Diagram <https://github.com/mattermost/docs/blob/broken-links/source/images/network.PNG>`__
```
Comment (lfbrock) [inline source/install/requirements.rst]: This link should probably go to master instead of broken-links branch

**Comment by lfbrock** [comment]
+1 once comments are addressed.

**Comment by esethna** [comment]
Updated

**Comment by esethna** [comment]
Holding off on merging this PR as it might cause conflicts with some larger community PRs being reviewed right now.

**Comment by jasonblais** [comment]
+1

**Pull request merged**